modules/oc-by-example-content.adoc

```diff
@@ -1858,6 +1858,12 @@ Log in to a server
 
   # Log in to the external OIDC issuer through Auth Code + PKCE by starting a local server listening port 8080
   oc login localhost:8443 --exec-plugin=oc-oidc --client-id=client-id --extra-scopes=email,profile --callback-port=8080
+
+  # Log in with an external OIDC
+  oc login --exec-plugin=oc-oidc --issuer-url=$ISSUER_URL --client-id=$CLI_CLIENT_ID --extra-scopes=email,profile --callback-port=8080
+
+  # Log in with an external OIDC if the external OIDC certificate is not publically trusted
+  oc login --exec-plugin=oc-oidc --issuer-url=$ISSUER_URL --client-id=$CLI_CLIENT_ID --extra-scopes=email --callback-port=8080 --oidc-certificate-authority router-ca/ca-bundle.crt
```

Comment:
```suggestion
  oc login --exec-plugin=oc-oidc --issuer-url=$ISSUER_URL --client-id=$CLI_CLIENT_ID --extra-scopes=email --callback-port=8080 --oidc-certificate-authority <CA for external OIDC certificate>
```

Reply:
It seems you'd better to not use the $VAR_NAME format in a document for the commands.

Reply:
BTW, does the document need to explain each argument --issuer-url/--client-id/--extra-scopes=email et al to end users?

Reply:
BTW --oidc-certificate-authority needs @ardaguclu's https://github.com/openshift/oc/pull/1720 implementation to be merged.

Reply:
If we wanted to explain each argument, I think we would want that in a different place in the docs. Maybe adding explanations here could be a future enhancement?

Reply:
Hi @arendej and @bchandra-ocp - Can I please get your input on how we want to explain the new usage? Thanks!